vagrant/php-overrides.ini

```diff
@@ -2,4 +2,4 @@
 
 expose_php = Off
 post_max_size = 250M
-upload_max_filesize = 200M
+upload_max_filesize = 200M
```

Comment:
It doesn't _really_ matter but leaving a newline at the end of a text file is a *nix convention (don't worry about changing it here - this is only an FYI). I have my IDE set to automatically leave them.